process_scripts/Fairness_Demos.R

```diff
@@ -0,0 +1,24 @@
+learner = lrn("classif.rpart", cp = .01)
```

Comment:
What is `process_scripts`. This is a non-standard folder. Please move the files to their appropriate destinations
(vignettes, data, ...)

Reply:
Sorry, that was added by accident, removed it.